### ISSUE TITLE
Consider using normalize.css as a dependency

### ISSUE THREAD

**Issue by necolas** [ianstormtaylor/reset#1]

**Comment by ianstormtaylor** [comment]
oooh nice didn't even realize it was a component, doh! gonna check that out tomorrow, sounds real good

**Comment by ianstormtaylor** [comment]
thanks! that cleaned up the code a lot

**Comment by yields** [comment]
@ianstormtaylor it's broken :|

``` text
error : failed to lookup "ianstormtaylor-reset"'s dependency "necolas-normalize-css"
```

cc @necolas

**Comment by yields** [comment]
https://github.com/necolas/normalize.css/pull/226

**Comment by ianstormtaylor** [comment]
shit wow, sorry!!! rolled back

**Comment by yields** [comment]
thanks! that was quick :)

**Comment by necolas** [comment]
Thanks. That wasn't part of the spec when TJ added the file to normalize.css.

Fixed in v2.1.3
